components/services/services.sass

```diff
@@ -32,7 +34,11 @@
   justify-content: space-evenly
   color: $service-box-text-color
   padding: 1rem
-  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.25) 
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.25)
```

Comment:
Was there a mixin we made for this?

Reply:
Yep, think it was in a different pull request but it's merged now so I'll apply